exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return (number < 0);
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return (number > 0 && number < 10);
```

Comment:
If statement is required here if the code find that the number is between 0 & 10 it will give expected result but when i give the numbers which are not between 0 & 10 it won't give me result